packages/app/components/feed/FeedCard.tsx

```diff
@@ -281,7 +281,7 @@ export default function Card({
                             gap: 8,
                           }}
                         >
-                          {user?.id === owner_id ? null : (
```

Comment:
A user should not be able to favorite their own stuff, do we need to remove this ?

Reply:
Okay, thought the other way around. I will be fixed.